### PR TITLE
FEA: add LightGCL

### PR DESCRIPTION
**Pull request by wending0417**
Add GeneralRecommender LightGCL to RecBole-GNN.

### CODE REVIEW

**Comment by wending0417** [comment]
作者们好~我主要在ml-1m数据集上验证LightGCL的性能。
超参数调整范围：
temp choice [0.2,0.5,0.8,2]
lambda1 choice [0.01,0.005,0.0001]
lambda2 choice [1e-5,1e-7]
dropout choice [0.1,0.0]
learning_rate choice [0.001,0.0005]

最优超参数设置：
dropout:0.1, lambda1:0.01, lambda2:1e-05, learning_rate:0.001, temp:2

最优结果：
Valid result:
recall@10 : 0.163    mrr@10 : 0.3629    ndcg@10 : 0.201    hit@10 : 0.7128    precision@10 : 0.149
Test result:
recall@10 : 0.1865    mrr@10 : 0.4273    ndcg@10 : 0.2473    hit@10 : 0.7377    precision@10 : 0.182

**Review by hyp1231 — APPROVED** [review]

**Comment by hyp1231** [comment]
Thanks for the great contribution! 🎉
